tests/test_verifylib.py

```diff
@@ -1023,6 +1037,20 @@ def test_verify_passing_double_signed_layout(self):
             [self.alice_path, self.bob_path]
         )
         in_toto_verify(layout, layout_key_dict)
+    
+    def test_verify_failing_layout_current_datetime(self):
+        with patch('datetime' + '.date') as mock_date:
```

Comment:
Consider what you want to be patching. The expiry check uses `datetime.datetime.now()`.

Reply:
Also, check that the mock holds through the in_toto_verify call.